src/helm/impress/values.yaml

```diff
@@ -48,32 +48,62 @@ ingress:
   ## @param ingress.customBackends Add custom backends to ingress
   customBackends: []
 
-## @param ingressWS.enabled whether to enable the Ingress or not
-## @param ingressWS.className IngressClass to use for the Ingress
-## @param ingressWS.host Host for the Ingress
-## @param ingressWS.path Path to use for the Ingress
-ingressWS:
+## @param ingressCollaborationWS.enabled whether to enable the Ingress or not
+## @param ingressCollaborationWS.className IngressClass to use for the Ingress
+## @param ingressCollaborationWS.host Host for the Ingress
+## @param ingressCollaborationWS.path Path to use for the Ingress
+ingressCollaborationWS:
   enabled: false
   className: null
   host: impress.example.com
-  path: /ws
+  path: /collaboration/ws/
   ## @param ingress.hosts Additional host to configure for the Ingress
   hosts: []
   #  - chart-example.local
-  ## @param ingressWS.tls.enabled Wether to enable TLS for the Ingress
-  ## @skip ingressWS.tls.additional
-  ## @extra ingressWS.tls.additional[].secretName Secret name for additional TLS config
-  ## @extra ingressWS.tls.additional[].hosts[] Hosts for additional TLS config
+  ## @param ingressCollaborationWS.tls.enabled Wether to enable TLS for the Ingress
+  ## @skip ingressCollaborationWS.tls.additional
+  ## @extra ingressCollaborationWS.tls.additional[].secretName Secret name for additional TLS config
+  ## @extra ingressCollaborationWS.tls.additional[].hosts[] Hosts for additional TLS config
   tls:
     enabled: true
     additional: []
 
-  ## @param ingressWS.customBackends Add custom backends to ingress
+  ## @param ingressCollaborationWS.customBackends Add custom backends to ingress
   customBackends: []
 
   annotations: 
+    nginx.ingress.kubernetes.io/auth-response-headers: "Authorization, X-Can-Edit, X-User-Id"
+    nginx.ingress.kubernetes.io/auth-url: https://impress.example.com/api/v1.0/documents/collaboration-auth/
     nginx.ingress.kubernetes.io/enable-websocket: "true" 
-    nginx.ingress.kubernetes.io/upstream-hash-by: "$request_uri"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "86400"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "86400"
+    nginx.ingress.kubernetes.io/upstream-hash-by: $arg_room
+
+## @param ingressCollaborationApi.enabled whether to enable the Ingress or not
+## @param ingressCollaborationApi.className IngressClass to use for the Ingress
+## @param ingressCollaborationApi.host Host for the Ingress
+## @param ingressCollaborationApi.path Path to use for the Ingress
+ingressCollaborationApi:
```

Comment:
I didn't understand the need for another full url range. To me, the collaboration server should only be available through urls that are checked by Django as this is the only way to compute access rights. Let's not create things that we don't need yet and see later when the problem arises?

Reply:
I need it,  I really try to not create a new ingress, but with Jacques it was what we found the cleanest.
First of all, the endpoints `/collaboration/api/` is callable only by the backend (django), we don't want clients to use them, the endpoints are secure with a key known only by the 2 servers.
Second things, we don't want the backend to pass by the auth_url `collaboration-auth`, `collaboration-auth` is made to secures clients access, the backend does not have notion of accesses or abilities, it does not make sense for the backend to call again himself, to get a key that it has already.

To avoid the auth_url `collaboration-auth`, we need to have another ingress.

I try first to server to server (django / yjs server), but it is working only aleatory, when the backend connect to the good pod. The backend must connect to the same pod as the client.
Then I try to create a ConfiMaps, like that I could indicate the stickiness, and depend the road avoid the auth_url `collaboration-auth`, but it is not the good way, the ConfigMaps is commun to other service than `Impress`, it can messed up things

If you have others things in mind, curious to hear about.

Reply:
see above comment. We should remove this ingress.